src/main/java/org/quiltmc/Constants.java

```diff
@@ -11,6 +11,7 @@ public final class Constants {
     static final String USER_AGENT = "update-quilt-meta/" + TOOL_VERSION;
 
     static final String GROUP = "org.quiltmc";
+    static final boolean TESTING = true;
```

Comment:
You probably wanted to set this to false, right?

Reply:
yes!